battleship.py

```diff
@@ -163,7 +182,12 @@ def isHorizontal(ship):
 Returns: list of ints
 '''
 def getClickedCell(data, event):
-    return
+    x = int(event.y/data["cell"])
+    y = int(event.x/data["cell"])
+    clickedCell = []
+    clickedCell.append(x)
+    clickedCell.append(y)

```

Comment:
you can create the list of x and y directly instead of creating an empty list and then appending

Reply:
Also, resolve conflicts by deleting pycache and then clicking on the resolve conflicts button for battleship.py